### PR TITLE
Feat/add pydynamicreporting

### PR DESCRIPTION
**Pull request by margalva**
Add cheat sheet for pydynamicreporting

### CODE REVIEW

**Comment by RobPasMue** [comment]
Hi @margalva! Thanks for opening this PR, really appreciated! I will try building it locally and give you feedback - reading LaTeX files is sometimes a bit obscure in terms of final result =)

File: cheat_sheets/pydynamicreporting_cheat_sheet/pydynamicreporting_cheat_sheet.tex [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+\item \href{https://dynamicreporting.docs.pyansys.com/version/stable/class_documentation.html}{\color{blue}{API reference}}}
+\item \href{https://dynamicreporting.docs.pyansys.com/version/stable/examples/index.html}{\color{blue}{Examples}}}
+\end{itemize}
+\end{multicols}
+
+% Footer session of the latex with link to documentation and GitHub page
+\vspace{-0.15cm}
+\noindent\makebox[\linewidth]{\rule{\paperwidth}{4pt}}
+\begin{center}
+Getting Started with PyAnsys \includegraphics[height=\fontcharht\font`\S]{slash.png} \href{https://github.com/pyansys}{\color{blue}{PyAnsys on GitHub}} \includegraphics[height=\fontcharht\font`\S]{slash.png} Visit \code{\href{https://dev.docs.pyansys.com/}}{\color{blue}{dev.docs.pyansys.com}}
```
Comment (Revathyvenugopal162) [inline cheat_sheets/pydynamicreporting_cheat_sheet/pydynamicreporting_cheat_sheet.tex:206]:
```suggestion
Getting Started with PyDynamicReporting \includegraphics[height=\fontcharht\font`\S]{slash.png} \href{https://github.com/ansys/pydynamicreporting}{\color{blue}{PyAnsys on GitHub}} \includegraphics[height=\fontcharht\font`\S]{slash.png} Visit \code{\href{https://dynamicreporting.docs.pyansys.com/}}{\color{blue}{dynamicreporting.docs.pyansys.com}}
```

**Comment by Revathyvenugopal162** [comment]
Thanks for opening the PR @margalva , can you please attach the rendered pdf of the cheatsheet in PR description or as comment for proof reading from docs team ? i will try to adapt black formatting in the script for this PR. adding @EstherProsperi for proof reading.

**Comment by margalva** [comment]
Please find the pdf attached here. Thank you!

> Thanks for opening the PR @margalva , can you please attach the rendered pdf of the cheatsheet in PR description or as comment for proof reading from docs team ? i will try to adapt black formatting in the script for this PR. adding @EstherProsperi for proof reading.

[pydynamicreporting_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12093070/pydynamicreporting_cheat_sheet.pdf)

**Comment by EstherProsperi** [comment]
[pydynamicreporting_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12093901/pydynamicreporting_cheat_sheet.pdf)

Embedded comments attached :)

**Comment by margalva** [comment]
> [pydynamicreporting_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12093901/pydynamicreporting_cheat_sheet.pdf)
> 
> Embedded comments attached :)

Thank you for the prompt feedback! New revisited pdf.
[pydynamicreporting_cheat_sheet.pdf](https://github.com/ansys/pyansys-cheat-sheet/files/12094269/pydynamicreporting_cheat_sheet.pdf)

**Review by EstherProsperi — APPROVED** [review]

**Comment by RobPasMue** [comment]
Thank you @EstherProsperi and @Revathyvenugopal162 for your comments - nothing to be added on top of their comments. Thanks @margalva for submitting this PR! Let's address the final comments from @EstherProsperi and merge!

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**